build/charts/flow-aggregator/values.yaml

```diff
@@ -18,6 +18,14 @@ flowAggregatorAddress: ""
 recordContents:
   # -- Determine whether source and destination Pod labels will be included in the flow records.
   podLabels: false
+# hostAliases defines custom host-to-IP mappings for a Pod.
+hostAliases:
+  # -- Determine whether to add a custom host-to-IP mapping to flow aggregator
+  enable: false
```

Comment:
there is no need for a boolean toggle here IMO

what we usually do in this case is have something like this in `values.yaml`:
```yaml
# -- HostAliases to be injected into the Pod's hosts file.
# For example: [{"ip": "8.8.8.8", "hostname": "foo.bar"}]
hostAliases: []
```

We do this when we expose a Pod spec property "as is" in the helm values (affinity, resources, labels, etc.)

Reply:
Thanks for reviewing and the suggestion. 
I've updated it.